draft-gruessing-moq-requirements.md

```diff
@@ -166,6 +173,13 @@ any member of the audience could participate. As this has a much larger total
 number of participants - as many as Live Media Streaming {{lmstream}}, but with
 the bi-directionality of conferencing, this should be considered a "hybrid". There can be additional functionality as well that overlap between the two, such as "live rewind", or recording abilities.
 
+Another consideration is the limits of "human bandwidth" - as the number of
+sources are included into a given session increase, the amount of media that can
+usefully understood by a single person diminishes. To put it more simply - too
+many people talking at once is much more difficult to understand than one person
+speaking at a time. Subsequently this will define some limitations in the number
+of concurrent or semi-concurrent, bidirectional communications that occur.
```

Comment:
Perhaps "**potential** concurrent or semi-concurrent, bidirectional communications that occur."?